index.html

```diff
@@ -1,10 +1,12 @@
-<!DOCTYPE html>
-<html>
-<head>
-   <meta http-equiv="refresh" content="0; url=http://devswag.io">
-</head>
-<body>
-   <p>Redirecting to: 
-   <a href="http://devswag.io">devSwag</a></p>
-</body>
-</html>
+<!DOCTYPE html>
+<html>
+<head>
+    <title>devSwag</title>
+    <link rel="icon" type="image/ico" href="C:\Users\GeethmakaDissanayake\Desktop\New folder (2)\swag-for-dev\site\src\img\logo.png" />

```

Comment:
This should be a relative path, something like `site/src/img/logo.png`.

Reply:
Oh sorry totally it's my fault